src/elmExecutor/buildPopulatedResourceBundle.js

```diff
@@ -3,6 +3,7 @@ function doSearch(smart, type, fhirVersion, request, callback) {
   const q = {};
   let usePatient = true;
   // setup the query for Practitioner and Coverage
+  // TODO - handle other resource not associated with Patient? 
```

Comment:
Is this part of another Jira ticket?

Reply:
Thinking to handle it in https://jira.mitre.org/browse/DMEERX-862. E.g. for Organization